lib/dradis/plugins/veracode/formats/flaw.rb

```diff
@@ -0,0 +1,19 @@
+module Dradis::Plugins::Veracode::Formats
+  module Flaw
+
+    private
+
+    def parse_flaw(xml_flaw)
+      cwe_id = xml_flaw[:cweid]
+      logger.info { "\t\t => Creating issue and evidence (flaw cweid: #{ cwe_id })" }
+
+      flaw = ::Veracode::Flaw.new(xml_flaw)
+      issue_text = mapping_service.apply_mapping(source: 'issue', data: flaw)
+      issue = content_service.create_issue(text: issue_text, id: cwe_id)
+
+      veracode_evidence = ::Veracode::Evidence.new(xml_flaw)
```

Comment:
Can we simply call this `evidence`?

Reply:
Might be confused with the model `Evidence`. The models (flaw, evidence) are also saved under `lib/veracode/*` while this class is under dradis/plugins/.. so we need to be explicit with the namespace here.

Reply:
I mean just this variable name